exercises/calculator.cpp

```diff
@@ -10,3 +10,19 @@
   Multiplication: 8
   Division: 2
 */
+#include <iostream>
+int main() {
+  using std::cout;
+  using std::cin;
+  int first;
+  int second;
+  cout << "Insert first number: ";
+  cin >> first;
+  cout << "Insert second number: ";
+  cin >> second;
+  cout << "Sum: " << first + second << "\n";
+  cout << "Difference: " << first - second << "\n";
+  cout << "Multiplication: " << first * second << "\n";
+  cout << "Division: " << first / second;
```

Comment:
Potresti spostare il codice in una funzione a parte

Reply:
Anche questa parte di codice l'ho modificata.